Dockerfile.test.gpu

```diff
@@ -123,7 +131,7 @@ RUN if [[ ${MPI_KIND} != "None" ]]; then \
 RUN if [[ ${TENSORFLOW_PACKAGE} != "tf-nightly-gpu" ]]; then \
         PROTOBUF_PACKAGE=""; \
         if [[ ${TENSORFLOW_PACKAGE} == tensorflow-gpu==1.15.* ]] || \
-           [[ ${TENSORFLOW_PACKAGE} == tensorflow-gpu==2.[012345].* ]]; then \
+           [[ ${TENSORFLOW_PACKAGE} == tensorflow-gpu==2.[0123456789].* ]]; then \
```

Comment:
Are you sure this is required for Tensorflow >= 2.6?

Reply:
I think this may have been leftover from prior experiments.  Just tried another image without this, which seemed to be OK, so will revert this line.